pixl_dcmd/tests/test_main.py

```diff
@@ -71,7 +71,7 @@ def _get_mri_diffusion_tags(
     so we can test whether the manufacturer overrides work during anonymisation
     """
     tag_ops = load_tag_operations(config)
-    mri_diffusion_overrides = tag_ops.manufacturer_overrides[0]
+    mri_diffusion_overrides = tag_ops.manufacturer_overrides[1]
```

Comment:
I've included `"mri.yaml"` in the manufacturer overrides for `test-extract-uclh-omop-cdm.yaml` (to be consistent with what's currently in `test-extract-uclh-omop-cdm-xnat.yaml` and `test-extract-uclh-omop-cdm-dicomweb.yaml`)